db/schema.rb

```diff
@@ -160,6 +160,10 @@
     t.index ["login"], name: "index_systems_on_login", unique: true
   end
 
+  add_foreign_key "activations", "services", on_delete: :cascade
```

Comment:
Should schema changes be really part of this PR?

Reply:
-> https://github.com/SUSE/rmt/pull/698